middleware.js

```diff
@@ -0,0 +1,15 @@
+// Custom middleware to turn username into uppercase
+
+const fixCase = (req, res, next) => {
+  const name = req.body.name;
+
+  if( name ){
+    req.body.name = name.charAt(0).toUpperCase() + name.slice(1);
+    // console.log( "name", req.body.name );
+    next();
+  } else {
+    next();
```

Comment:
You have `next()` in both statements - since it is getting called twice, how about removing the else and putting `next()` outside of the if statement?

Reply:
Fixed! Thank you!